examples/gameroom/daemon/src/authorization_handler/mod.rs

```diff
@@ -1323,7 +1326,7 @@ mod test {
     fn get_new_gameroom_service(circuit_id: &str, timestamp: SystemTime) -> NewGameroomService {
         NewGameroomService {
             circuit_id: circuit_id.to_string(),
-            service_id: "scabbard_123".to_string(),
+            service_id: "gr00".to_string(),
```

Comment:
should these be gm based on the previous commit?

Reply:
Fixed previous commit (should be `gr`)